pkg/kubelet/pod_workers.go

```diff
@@ -819,6 +857,44 @@ func (p *podWorkers) acknowledgeTerminating(pod *v1.Pod) PodStatusFunc {
 	return nil
 }
 
+// queueReservedWork queues a reserved work having the same full name as pod
```

Comment:
Can you add the why here? - It won't necessarily be immediately obvious to future readers of this code.